src/common/image.c

```diff
@@ -680,7 +680,7 @@ gboolean dt_image_get_final_size(const int32_t imgid, int *width, int *height)
   const gboolean use_raw =
     !dt_conf_is_equal("plugins/lighttable/thumbnail_raw_min_level", "never");
 
-  if(!img.verified_size && !dt_image_altered(imgid) && !use_raw && !incompatible)
```

Comment:
`verified_size` field is here to avoid to do all the rawspeed things each time...

What happen is that for some RAW file, we can't count on `img.width / height` to get the image dimensions as what is stored at first here (for some RAW) is the preview image dimensions which is lower than the real size. We have faced this on lighttable zooming.
But this rawspeed pass has only to be done one time, then` img.width/height` are correct.
With your change I fear that we loose some prefs each time we want to recalculate the final dimension and `img.final_width/height `have been resetted by `dt_image_reset_final_size`

Reply:
Mmh,
a few lines above we check for `if(img.final_height > 0 && img.final_width > 0)` and return immediately if these are valid resp >0.

Aah, maybe i understand, the verified flag is just to have
```
    imgtmp = dt_image_cache_get(darktable.image_cache, imgid, 'w');
    dt_imageio_open(imgtmp, filename, NULL);
```
filling the database with some data we want to calculate just once?